Resources/public/js/views/serverside/ez-roleserversideview.js

```diff
@@ -82,5 +84,23 @@ YUI.add('ez-roleserversideview', function (Y) {
         _uiUnsetAssignRoleLoading: function (button) {
             button.removeClass('is-loading').set('disabled', false);
         },
+
+        /**
+         * Refreshes the role view, showing the assignments tab.
+         *
+         * @method _refreshAssignmentsTab
+         * @protected
+         * @param {Y.Node} button
+         */
+        _refreshAssignmentsTab: function (button) {
+            /**
+             * Fired when the view needs to be refreshed.
+             * @event refreshAssignmentsTab
+             * @param roleId {String} The ID of the role we are showing assignments for
+             */
+            this.fire('refreshAssignmentsTab', {
```

Comment:
Please stop calling everything "magic", yeah maybe we need to add some code, that's our job. And no, no follow up here, we should just try to not duplicate code.